guides/source/active_record_migrations.md

```diff
@@ -246,7 +246,7 @@ class AddUserRefToProducts < ActiveRecord::Migration[5.0]
 end
 ```
 
-This migration will create a `user_id` column and appropriate index.   
+This migration will create a `user_id` column and appropriate index.
```

Comment:
Seeing these in many past changes. This skews commit history, and hard to debug issues.